src/test/java/de/tum/in/www1/artemis/AbstractSpringIntegrationTest.java

```diff
@@ -42,4 +43,7 @@
 
     @SpyBean
     protected WebsocketMessagingService websocketMessagingService;
+
+    @SpyBean
+    protected CourseService courseService;
```

Comment:
is this really necessary? I would like to avoid this, because this would mean all tests use the courseService as SypBean

Reply:
@krusche Yes, you are right. I can add the SpyBean only to the `ModelingExerciseIntegrationTest`